corehq/apps/sso/forms.py

```diff
@@ -271,6 +272,14 @@ def __init__(self, identity_provider, *args, **kwargs):
             args=(identity_provider.owner.id,)
         )
 
+        if self.idp.is_editable:
+            self.fields['is_editable'].help_text = mark_safe(
+                '<a href="{}">{}</a>'
+            ).format(
+                utils.get_dashboard_link(self.idp),
+                _("Edit Enterprise Settings")
+            )
```

Comment:
it's definitely still used! I hear you on later deleted code bit, but I don't believe that's the case here. I think github collapsed the `forms.py` diff in the summary view because it was quite large.

Reply:
It's not that `mark_safe` is not used, it's just not a best practice to use it that way.

Reply:
sorry this was meant for the other comment. I'll update :)